frontstage/views/secure_messaging/create_message.py

```diff
@@ -77,3 +77,26 @@ def send_message(party_id, is_draft, case_id, survey, ru_ref):
     logger.info('Secure message sent successfully',
                 message_id=sent_message['msg_id'], party_id=party_id)
     return sent_message
+
+
+def create_headers():
+    encoded_jwt = SessionHandler().get_encoded_jwt(request.cookies['authorization'])
+    headers = {"jwt": encoded_jwt}
+    return headers
+
+
+def get_message(message_id, label, party_id):
+    logger.debug('Attempting to retrieve message', message_id=message_id, party_id=party_id)
+
+    headers = create_headers()
+    endpoint = 'secure-messaging/message'
+    parameters = {"message_id": message_id, "label": label, "party_id": party_id}
+    response = api_call('GET', endpoint, parameters=parameters, headers=headers)
+
+    if response.status_code != 200:
```

Comment:
I partially agree with this solution. I think the 200 range shouls always be sucsessfull. One day we should start to create some differentiation about the possible reposnse. http://www.flaskapi.org/api-guide/status-codes/ But I think it s ok for now.

Reply:
Agreed, I think this is code that was already there so until we figure out what should be used (my vote is for raise_for_status()), I'll leave it alone as it works.